app/controllers/api/maintenance_stats_controller.rb

```diff
@@ -3,6 +3,11 @@ class Api::MaintenanceStatsController < Api::ApplicationController
 
   def enqueue
     @project.update_maintenance_stats_async(priority: :high)
-    head :ok
+    head :accepted
+  end
+
+  def begin_watching
+    @project.update_maintenance_stats_async(priority: high) if @project.repository_maintenance_stats.length == 0
```

Comment:
Can we use `empty?` here instead of `length == 0`?

Reply:
Or maybe `exists?`. I think that is supposed to be the most efficient version of this.